README.md

```diff
@@ -143,7 +143,7 @@ rules:
 - apiGroups:
   - aws.k8s.logmein.com
   resources:
-  - eips
+  - eips, eipassociations
```

Comment:
This will not work, you need:
```yaml
resources:
- eips
- eipassociations
```

Reply:
oh right in my test pod, I changed it but not here. Good catch